### ISSUE TITLE
Source Repositories are empty

### ISSUE THREAD

**Issue by RoganDawes** [aurgatech/issues#6]
While Aurga has benefitted from various open source projects such as U-boot and the Linux kernel, and have created a repo to hold their changes to the Linux kernel, that repository is empty (and there is nothing for u-boot).

Please can this be addressed soonest? Thank you.

**Comment by aurgatech** [comment]
Yes, We'll are preparing this. Since we are having some urgent things to process. We will release the code several weeks later.

**Comment by RoganDawes** [comment]
It has been at least 4 months since the empty kernel repo was created, judging by the 4 month old issue.

Being "busy" doesn't absolve you from legal obligations.

**Comment by RoganDawes** [comment]
Repos have been populated, thank you. Very disappointing that you chose to publish with no history, though, making it difficult to determine what changes you have made.
